includes/class-sensei-teacher.php

```diff
@@ -278,6 +278,28 @@ public function teacher_meta_box_content( $post ) {
 
 		<?php
 
+		// Get `sensei_teacher_meta_box_coteachers_feature` contents.
+		ob_start();
+		do_action( 'sensei_teacher_meta_box_coteachers_feature', $post );
```

Comment:
I don't really like having an action that mentions a specific feature like this. I would prefer an action that is more generic (perhaps `sensei_teacher_meta_box_after`?) and an explicit hook for hiding the upsell (perhaps `sensei_course_coteachers_hide` in keeping with our other [other upsell filters](https://github.com/Automattic/sensei/blob/5f35d440c6ecbf7f5250f8868bd616e85b40389f/includes/class-sensei-lesson.php#L292)?).

Reply:
Done: [571a998](https://github.com/Automattic/sensei/pull/5345/commits/571a998d7558a854b5ae7a0fed983300e42ffe2f).